src/service/private.overseas.entity.details.ts

```diff
@@ -3,34 +3,30 @@ import { Request } from "express";
 import { makeApiCallWithRetry } from "./retry.handler.service";
 import { createAndLogErrorRequest, logger } from "../utils/logger";
 import { OverseasEntityExtraDetails } from "@companieshouse/api-sdk-node/dist/services/overseas-entities/types";
-import { ApplicationData } from "../model/application.model";
-
-export const hasRetrievedPrivateOeDetails = (appData: ApplicationData): boolean =>
-  !!appData.entity?.email;
 
 export const getPrivateOeDetails = async (
   req: Request,
-  oeNumber: string,
+  transactionId: string,
+  overseasEntityId: string,
 ): Promise<OverseasEntityExtraDetails | undefined> => {
   const response = await makeApiCallWithRetry(
     "overseasEntity",
     "getOverseasEntityDetails",
     req,
     req.session as Session,
-    oeNumber,
+    transactionId,
+    overseasEntityId,
   );
 
   if (response.httpStatusCode !== 200 && response.httpStatusCode !== 404) {
-    const errorMsg = `Something went wrong fetching company details = ${JSON.stringify(response)}`;
+    const errorMsg = `Something went wrong fetching overseas entity details = ${JSON.stringify(response)}`;
     throw createAndLogErrorRequest(req, errorMsg);
   }
 
   if (response.httpStatusCode === 404) {
-    logger.debugRequest(req, `No company data found for ${oeNumber}`);
+    logger.debugRequest(req, `No overseas entity details found`);
```

Comment:
```suggestion
    logger.debugRequest(req, 'No overseas entity details found');
```

Unless it's worth adding the `transactionId` or `overseasEntityId` to the log message